providers/catalog.go

```diff
@@ -859,4 +860,33 @@ var catalog = CatalogType{ // nolint:gochecknoglobals
 			Write:     false,
 		},
 	},
+
+	// Slack configuration
+	Slack: {
+		AuthType: Oauth2,
+		BaseURL:  "https://api.slack.com",
```

Comment:
From https://api.slack.com/methods/conversations.acceptSharedInvite and other methods, I can see that the base URL is something different, would be good to understand if these are perfect aliases or there's any differences

Reply:
Looks like both of the URLs can be used interchangeably. Screenshots attached. For clarity, I will update the  base URL to reflect the documentation and will make a note on Slab.
![Screenshot 2567-04-08 at 11 40 44](https://github.com/amp-labs/connectors/assets/103050835/2062b7c8-267b-42c5-94b2-4d3e3c6b1195)
![Screenshot 2567-04-08 at 11 41 09](https://github.com/amp-labs/connectors/a
![Screenshot 2567-04-08 at 12 15 27](https://github.com/amp-labs/connectors/assets/103050835/27fb6a54-1bfd-48ff-adb8-974907beef18)
![Screenshot 2567-04-08 at 12 15 06](https://github.com/amp-labs/connectors/assets/103050835/52607ddd-4b53-471b-8c58-cab2873a07c9)
ssets/103050835/10785f23-a3b1-420d-aa34-f63f89ec0c0c)

Reply:
Thank you!

Reply:
Updated. When using https://slack.com/api, no need to use localhost:4444/api, just localhost:4444/{{method}}